### DISCUSSION TITLE
Release Roadmap for 'info.picocli:picocli-shell-jline3:*'

### DISCUSSION THREAD

**Discussion by amarktl** [remkop/picocli#2345] [General]
Hi, 

First of all - Thank you for this great library. :)

We are using it heavily and are very happy with it.  Currently we are facing some issues with the resizing of the JLine Terminal.

We are hoping that some of this things get better with newer versions. 

When are you planning to release a new version with updated dependencies of the picocli / jline3 integration ( https://central.sonatype.com/artifact/info.picocli/picocli-shell-jline3 ) ?

BR,
Alex

**Comment by remkop** [comment]
Hi @amarktl thank you for your message!
Have you tried using picocli-shell-jline3 with the latest version of JLine (I think that is 3.27.1),
and did it work?

I have not tried yet, it would be very helpful if you could give it a try and let me know if any changes are required in picocli-shell-jline3 in order to upgrade the dependency to Jline 3.27.1.

**Comment by amarktl** [comment]
Hi - i will try and report back. :) 
BR
